### ISSUE TITLE
Width of widget

### ISSUE THREAD

**Issue by xSOADx** [edadams/themes#3]
Hi,
I installed your awesome theme but I have a problem
I found that width of Spotify is too wide and I don't know how to fix it
In official Win98 theme there were no problems with it
Could you help me please?
![image](https://cloud.githubusercontent.com/assets/8180837/21938693/f36b336c-d9ee-11e6-9f14-90f5fbc11ead.png)
http://sergeyrakhmalevich.tumblr.com

**Comment by edadams** [comment]
Hey there.

Unfortunately I don't still support the Win98 theme and if it's not working
then I'm not really able to do much to help you.

Sorry about that.

**Comment by xSOADx** [comment]
But maybe just advice about code? Something?

**Comment by edadams** [comment]
If it was that simple, I would have done it. The issue is the images that I took (from a real installation of Windows 98 on an ancient laptop) are of a fixed width. The width of Spotify's widget is different, and it is not a straight forward process to resize it down.

One solution is for me to remake the images of Windows 98 for the new widget size. This is possible but **extremely** time consuming. An alternative is to shrink the Spotify widget to the size currently in the theme. This is difficult.

I understand your unhappiness, but perhaps I can remind you you're using a theme that is at least 2 or 3 years old. I've moved on from this, I have to make money and support myself. I cannot invest the time and effort to fix this.
